### ISSUE TITLE
Installation wizard - Alerts Setup 1 (Info section)

### ISSUE THREAD

**Issue by crsfalzon** [SimplyVC/panic#181]
### Story
As a node operator, when I'm done with repository setup, I want the ability to review and update a pre-configured collection of out-of-the-box threshold alerts and alert severities, so that if needed, I can easily and quickly customise these to better fit my monitoring needs.
### Description
The scope of this ticket is limited to the first section of the Alerts setup page as shown in Image 1. Here the node operator is provided with the rationale while being educated about this step in the setup process. 

Image 1
![Alerts setup](https://user-images.githubusercontent.com/89395527/166101692-3fca7396-50ef-4c4c-a2d1-4a64900c8d39.PNG)
### Requirements:
- Page is displayed at /installation/alerts
- Ticket includes the completion of headers and footers as found in previous installation steps.
- Display vertically stacked and centred the Alerts setup logo, the text "Alerts setup for _**sub-chain name**_", and the progress in the setup process as a whole in fraction form (i.e. **step 5/5**). Note that the sub-chain name would have been previously inputted by the node operator. (See Image 1)
- Replicate the text and icons shown in Image 1, with the exception of the placement of the info button. In the text, "... critical threshold and time_windows, otherwise...", omit the underscore and replace with a space.
- Clicking on the info button placed next the the text, "PANIC supports four types of alerts:" triggers the info modal. Replicate the structure and content as shown in Images 2 and 3.
- Clicking on the "Got it!" button closes the modal (see Image 3).

Image 2
![Alerts setup info popup1](https://user-images.githubusercontent.com/89395527/166102026-eb906c7e-b37e-4b22-bd81-bd1839d955bc.PNG)

Image 3
![Alerts setup info popup2](https://user-images.githubusercontent.com/89395527/166102034-48deb9ca-0653-45d1-8ee4-3abd3426ce3b.PNG)
### Blocked by
Not dependent on another ticket to be tackled.
### Acceptance Criteria
**Scenario**: Alerts setup, info section
**Given**: The node operator has completed his repositories setup
**When**: He clicks on the Next button
**Then**: He is presented with the Alerts setup page, displaying the step's rationale and information about the alert types (see Image 1)
**And**: If the node operator clicks on the info button
**Then**: He is presented with the info modal populated with the content displayed in Images 2 and 3
**And**: If the node operator clicks on the "Got it!" button
**Then**: The modal is closed and he can review again the info being presented
